src/CaptureClient/include/CaptureClient/ClientCaptureOptions.h

```diff
@@ -29,6 +29,14 @@ struct ClientCaptureOptions {
   orbit_grpc_protos::CaptureOptions::UnwindingMethod unwinding_method =
       orbit_grpc_protos::CaptureOptions::UnwindingMethod::CaptureOptions_UnwindingMethod_kUndefined;
 
+  orbit_grpc_protos::CaptureOptions::ThreadStateChangeCallStackCollection
+      thread_state_change_callstack_collection =
+          orbit_grpc_protos::CaptureOptions::kThreadStateChangeCallStackCollectionUnspecified;
+
+  orbit_grpc_protos::CaptureOptions::UnwindingMethod
```

Comment:
I don't think we should offer a different unwinding method setting other than the "overall" one that is used for sampling.
It makes thinks easier for the user and also simplifies the code.

Reply:
What do you think @beckerhe ?

Reply:
I agree. I also thought we concluded this will be only for testing purposes and is going to be removed later.